### PR TITLE
ECM-616

### PR DESCRIPTION
**Pull request by hsjhita**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-616

### Change description ###

Code change so Judge's ITCO reference doesn't show

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java [hunk 1]
```diff
@@ -177,8 +178,7 @@ private void addReportSummary(HearingsToJudgmentsReportSummary reportSummary,
                 hearingJudgmentItem.judgmentDateSent = dateJudgmentSent.format(OLD_DATE_TIME_PATTERN2);
                 hearingJudgmentItem.total = hearingListedDate.datesUntil(dateJudgmentSent).count();
                 hearingJudgmentItem.reservedHearing = dateListedType.getHearingReservedJudgement();
-                hearingJudgmentItem.judge = hearingType.getJudge();
-
+                hearingJudgmentItem.judge = hearingType.getJudge().substring(hearingType.getJudge().indexOf('_') + 1);
```
Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java:181]: May be this looks a bit cleaner?
Strings.split(hearingType.getJudge(), '_')[1]

Comment (gauravchawla73463) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java:181]: May be this looks a bit cleaner?
Strings.split(hearingType.getJudge(), '_')[1]

**Review by gauravchawla73463 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java]: Yh, may be you can leave it as it was or use this :) 

String[] segments = Strings.split(hearingType.getJudge(), "_");
                hearingJudgmentItem.judge = segments[segments.length - 1];

Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java]: Will see what other reviewers say

**Review by btensay — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java (hunk 1 above)
Comment (btensay) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java]: I think checking for null is the simplest and more readable way.

Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingstojudgments/HearingsToJudgmentsReport.java]: Added a null check and small refactor

**Pull request merged**